# GATK-CaseStudy

GATK Variant Calling Workflow using Galaxy, CWL and Cpipe
The data used in this case study is provided via a cloud container as detailed below becasue of size limitation of git repository. 

## Data Availability: 
1. The data used in this analysis can be downloaded from the Cloud container: https://swift.rc.nectar.org.au:8888/v1/AUTH_377/Paper-Data.
2. This container will display the directory structure and names of all the files included in it such as: 
   1. InputData:
      * Fastq files for chromosome 21:
        1. FQ_1.fastq (https://swift.rc.nectar.org.au:8888/v1/AUTH_377/Paper-Data/InputData/FQ_1.fastq)
        2. FQ_2.fastq (https://swift.rc.nectar.org.au:8888/v1/AUTH_377/Paper-DataInputData/FQ_2.fastq)
      * Indel and SNP calls:
        1. 1000G_phase1.indels.hg19.vcf (https://swift.rc.nectar.org.au:8888/v1/AUTH_377/Paper-Data/InputData/1000G_phase1.indels.hg19.vcf)
        2. Mills_and_1000G_gold_st.vcf (https://swift.rc.nectar.org.au:8888/v1/AUTH_377/Paper-Data/InputData/Mills_and_1000G_gold_st.vcf)
        3. dbsnp_138.vcf (https://swift.rc.nectar.org.au:8888/v1/AUTH_377/Paper-Data/InputData/dbsnp_138.vcf)
      * Human Reference sequence with the index files (generated by using bwa index â€“a bwtsw hg19.fasta):
        1. hg19.fasta (https://swift.rc.nectar.org.au:8888/v1/AUTH_377/Paper-Data/InputData/hg19.fasta)
        2. hg19.fasta.amb (https://swift.rc.nectar.org.au:8888/v1/AUTH_377/Paper-Data/InputData/hg19.fasta.amb)
        3. hg19.fasta.ann (https://swift.rc.nectar.org.au:8888/v1/AUTH_377/Paper-Data/InputData/hg19.fasta.ann)
        4. hg19.fasta.bwt (https://swift.rc.nectar.org.au:8888/v1/AUTH_377/Paper-Data/InputData/hg19.fasta.bwt)
        5. hg19.fasta.pac (https://swift.rc.nectar.org.au:8888/v1/AUTH_377/Paper-Data/InputData/hg19.fasta.pac)
        6. hg19.fasta.sa (https://swift.rc.nectar.org.au:8888/v1/AUTH_377/Paper-Data/InputData/hg19.fasta.sa)
      * Truthset and interval bed file:
        1. broadinstitute-human-exomechr21.bed (https://swift.rc.nectar.org.au:8888/v1/AUTH_377/Paper-Data/InputData/broadinstitute-human-exomechr21.bed)
        2. chr21-truthset-annotated.recode.vcf (https://swift.rc.nectar.org.au:8888/v1/AUTH_377/Paper-Data/InputData/chr21-truthset-annotated.recode.vcf
        3. NA12878-Truthset.vcf (https://swift.rc.nectar.org.au:8888/v1/AUTH_377/Paper-Data/InputData/NA12878-Truthset.vcf)
      * Resultant variant (vcf) files
        1. Galaxy workflow - Galaxy-broad-filtered.recode.vcf (https://swift.rc.nectar.org.au:8888/v1/AUTH_377/Paper-Data/Results/Galaxy-broad-filtered.recode.vcf)
        2. Cpipe workflow - cpipe-broad-filtered.recode.vcf (https://swift.rc.nectar.org.au:8888/v1/AUTH_377/Paper-Data/Results/cpipe-broad-filtered.recode.vcf)
        3. CWL workflow - cwl-broad-filtered.recode.vcf (https://swift.rc.nectar.org.au:8888/v1/AUTH_377/Paper-Data/Results/cwl-broad-filtered.recode.vcf)


